locales/de.json

```diff
@@ -1811,30 +1811,44 @@
 			"text": {
 				"eachTopicSeparate": " Bitte sende jedes Thema einzeln ab.",
 				"chooseArea": "Bereich wählen",
-				"pDashboard": "Übersicht",
+				"pDashboard": "Dashboard",
 				"pCourses": "Kurse",
```

Comment:
this will lead to confusion.... We call the page Übersicht (also from nuxt)

Reply:
Well yes, but components doesn't https://docs.hpi-schul-cloud.org/pages/viewpage.action?pageId=154176580

Reply:
Nevertheless, I will ask Kai about that.